Libplanet/State/AccountStateDeltaImpl.cs

```diff
@@ -67,10 +68,22 @@ Address signer
                     g => (IImmutableSet<Currency>)g.Select(kv => kv.Key.Item2)
                 .ToImmutableHashSet());
 
+        /// <inheritdoc/>
+        IImmutableDictionary<Address, IImmutableSet<Currency>>
+            IAccountStateDelta.TotalUpdatedFungibleAssets => TotalUpdatedFungibles
+                .GroupBy(kv => kv.Key.Item1)
+                .ToImmutableDictionary(
+                    group => group.Key,
+                    group => (IImmutableSet<Currency>)group.Select(kv => kv.Key.Item2)
+                .ToImmutableHashSet());
```

Comment:
Knitpick but this indent might be misleading because it's on the same level as `.ToImmutableDictionary()`.